docs/source/data/kedro_dvc_versioning.md

```diff
@@ -0,0 +1,294 @@
+```
+
+Then we can track the changes with DVC, and commit them to Git:
+
+```bash
+dvc add data/01_raw/companies.csv
+git add data/01_raw/companies.csv.dvc
+git commit -m "Track dataset changes with DVC"
+```
+
+DVC integrates with Git to manage different dataset versions. If you need to restore a previous version of a dataset, first identify the commit containing the desired version. You can use:
+
+```bash
+git log -- data/01_raw/companies.csv.dvc
+```
+
+To display the commit hashes associated with this file. Once you find the desired commit, run:
+
+```bash
+git checkout <commit_hash> data/01_raw/companies.csv.dvc
+dvc checkout
+```
+
+The first command will restore the `.dvc` metadata file to its previous version. The second uses the metadata file to restore the corresponding dataset.
+
+```bash
+Building workspace index
+Comparing indexes
+Applying changes
+M       data/01_raw/companies.csv
+```
+
+Using the command `tail data/01_raw/companies.csv` again shows that the dataset file has been restored to a previous version.
+
+```bash
+1618,100%,,1.0,t
+6957,,Rwanda,1.0,t
+7554,100%,,1105.0,f
+34243,95%,Uzbekistan,1.0,f
+12502,89%,Denmark,1.0,f
+20213,,Russian Federation,1.0,f
+2235,100%,Guinea,1.0,f
+2353,100%,Senegal,2.0,t
+49772,100%,Jersey,1.0,f
+16548,90%,,2.0,f
+```
+
+## Advanced use cases
+
+### How to store data remotely
+
+DVC remotes provide access to external storage locations to track and share your data and ML models with the `dvc push` and `dvc pull` commands. Those will be shared between devices or team members who are working on a project. It supports several different storage types, like Amazon S3, Azure Blob Storage or Google Cloud Storage, as well as self-hosted options. For more detail on this subject, see the [DVC documentation on remote storage](https://dvc.org/doc/user-guide/data-management/remote-storage#supported-storage-types).
+
+For example:
+
+```bash
+dvc remote add myremote s3://mybucket
+kedro run
+git add .
+git commit -m "Update"
+dvc push
+```
+
+### How to go back to a previous version of the data, stored remotely
+
+```bash
+git checkout <commit hash> data/01_raw/companies.csv.dvc
+dvc checkout
+dvc pull
+```
+
+### How to version with DVC data pipelines
+
+While the previous method allows you to version datasets, it comes with some limitations, as DVC requires the files to be tracked to be added manually:
+
+- Intermediate and output datasets must be added to DVC manually.
+- Parameters and code changes are not explicitly tracked.
+- Artifacts and metrics can be cumbersome to track.
+
+To address these issues, you can define Kedro pipelines as DVC stages in the dvc.yaml file. The list of stages is typically the most important part of a dvc.yaml file. The file can also be used to configure artifacts, metrics, parameters, and plots, either as part of a stage definition or on their own.
+
+### How to define Kedro pipelines as DVC stages
+
+Here is an example configuration for dvc.yaml:
```

Comment:
A brief description of what the `cmd`, `outs` and `deps` keys mean here would be nice!

Reply:
Added a link to the docs page that explains those in detail.